website/pages/docs/configuration/storage/raft.mdx

```diff
@@ -85,4 +85,49 @@ set `ha_enabled = true`.
   snapshot. Servers may take longer to recover from crashes or failover if this
   is increased significantly as more logs will need to be replayed.
 
+- `retry_join` `(list: [])` - There can be one or more `retry_join` stanzas.
+When the raft cluster is getting bootstrapped, if the connection details of all
+the nodes are known beforehand, then specifying this config stanzas enables the
+nodes to automatically form a raft cluster. All the nodes would mention all
```

Comment:
You may also want to mention that the node still needs to be unsealed if using shamir?

Reply:
Good point. Done.